src/pfe/file-watcher/server/src/projects/projectUtil.ts

```diff
@@ -440,6 +440,17 @@ async function executeBuildScript(operation: Operation, script: string, args: Ar
                         odoProjectInfo.compositeAppName = appName;
                         await projectsController.saveProjectInfo(projectID, odoProjectInfo, true);
                     }
+                    else if (operation.projectInfo.projectType != "appsody") {
```

Comment:
Just a small reminder here, we can treat appsody the same (if all goes as planned)

Reply:
Want me to change it now, or after the appsody projectID change is delivered?

Reply:
Don't want to break appsody on Kube if the needed change in Appsody isn't there yet.

I can open another PR once the CLI is updated

Reply:
Can change it now if not finding the appropriate service is handled gracefully

Reply:
KK, fixed